llvm/lib/Target/AMDGPU/AMDGPUMachineFunction.cpp

```diff
@@ -19,6 +19,26 @@
 
 using namespace llvm;
 
+static const GlobalVariable *
+getKernelDynLDSGlobalFromFunction(const Function &F) {
+  const Module *M = F.getParent();
+  std::string KernelDynLDSName = "llvm.amdgcn.";
```

Comment:
SmallString

Reply:
Updated to SmallString in latest commit.